src/features/rewards/welcomePage/__snapshots__/spec.tsx.snap

```diff
@@ -15,7 +15,7 @@ exports[`WelcomePage tests basic tests matches the snapshot 1`] = `
 .c0 {
   background: #f2f4f7;
   min-height: 100vh;
-  width: 100%;
+  min-width: 1024px;
```

Comment:
Note: This enforces the same behavior on the desktop welcome page, which seems desirable as we use the same SettingsPage component. This will **not** however, do so for the mobile welcome page, as it does not render inside the SettingsPage component.